src/components/MenuLink/index.tsx

```diff
@@ -35,7 +21,26 @@ MenuLink.propTypes = {
    * - `import { Link } from 'react-router'`
    * - `Link={Link}`
    */
-  Link: PropTypes.elementType,
-};
+  Link?: React.ElementType;
+  href?: string;
+}
+
+const MenuLink = ({
+  onSelect,
+  children,
+  isDisabled,
+  Link,
+  ...rest
+}: MenuLinkProps): JSX.Element => (
+  <ReachMenuLink
+    className={isDisabled ? 'fdsMenuItem--disabled' : undefined}
+    onSelect={!isDisabled ? onSelect : noop}
+    aria-disabled={isDisabled}
+    as={Link}
```

Comment:
having trouble with this, old jsx file had `Link` as optional and so does a story, but `ReachMenuLink` doesn't like it when i pass a Link that can be undefined.

Reply:
If you pass undefined into `as`, will it work? It didn't seem like it from what I saw.

Reply:
I would just define `Link` as `any` here . The definition of that prop is pretty complicated from what I saw.

Reply:
yeah, plus it doesn't make sense what the library expects, it says `as: 'a' | undefined`, and in their docs it says `string | Component` where component uses the `React.forwardRef` API.

normally i'm against using `any` but this in fact looks complicated and poorly documented.